### PR TITLE
Komiks

### PR DESCRIPTION
**Pull request by Komiks8457**
added email template support.
fixed the return -1 makes gateway do the endless loop dance.
added email option CheckCertificationRevocation its true by default but in my case it works with false.

### CODE REVIEW

File: ISRORBilling.csproj [hunk 1]
```diff
@@ -36,4 +43,7 @@
     <ItemGroup>
       <Folder Include="Services\Notification\CommunityProvided" />
     </ItemGroup>
+    <Target Name="PostBuild" AfterTargets="PostBuildEvent">
+      <Exec Command="XCOPY &quot;$(TargetDir)$(TargetName).dll&quot; &quot;H:\SILKROADR\SRO_Server\Billing\&quot; /F /Y&#xD;&#xA;XCOPY &quot;$(TargetDir)$(TargetName).exe&quot; &quot;H:\SILKROADR\SRO_Server\Billing\&quot; /F /Y" />
```
Comment (devtekve) [inline ISRORBilling.csproj:47]: I don't think this will work on other's computers

File: Template/SampleTemplate.html [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    <meta name="Keywords" content="silkroad, silkroadonline, Joymax, Wemade Max, onlinesilkroad, silkroad-online" />
+    <title>%ServerName%</title>
+</head>
+<body>
+    <table width="683" align="center" border="0" cellpadding="0" cellspacing="0">
+        <col width="31" />
+        <col width="622" />
+        <col width="30" />
+        <tr>
+            <td colspan="3"><img src="http://img.joymax.com/property/joymax/joymax_ml/us/authentication/top.jpg" alt="Wemade Max" /></td>
```
Comment (devtekve) [inline Template/SampleTemplate.html:14]: I am worried this might be too much abuse with joymax xD

File: ISRORBilling.csproj (hunk 1 above)
Comment (Komiks8457) [reply ISRORBilling.csproj:47]: oh pls remove that xd hahaha

File: Template/SampleTemplate.html (hunk 2 above)
Comment (Komiks8457) [reply Template/SampleTemplate.html:14]: well im sure most poeple will not use that template.. anyway thats just a sample

Comment (Neemo07) [reply Template/SampleTemplate.html:14]: DevTek , I think every developer should be smart to use it and its already Null by default so who will use it should use his customized template , I tested it and its really good option to be added ..

Regards

**Pull request closed without merging**